### PR TITLE
Add `debugEnabled` option for run and test

### PR DESCRIPTION
**Pull request by kateinoigakukun**
When we use some libraries like [johnno1962/Fortify](https://github.com/johnno1962/Fortify), we need to disable debugger.
So I added the option which enables us to disable it explicitly.

### CODE REVIEW

**Comment by yonaskolb** [comment]
I tested this by editing a scheme in Xcode and noticed that when deselecting `debug executable` that the `selectedLauncherIdentifier` also changed from `Xcode.DebuggerFoundation.Launcher.LLDB` to `Xcode.IDEFoundation.Launcher.PosixSpawn`

File: Docs/ProjectSpec.md [hunk 1]
```diff
@@ -687,6 +687,7 @@ The different actions share some properties:
 - [ ] **disableMainThreadChecker**: **Bool** - `run` and `test` actions can define a boolean that indicates that this scheme should disable the Main Thread Checker. This defaults to false
 - [ ] **language**: **String** - `run` and `test` actions can define a language that is used for Application Language
 - [ ] **region**: **String** - `run` and `test` actions can define a language that is used for Application Region
+- [ ] **debugEnabled**: **Bool** - `run` and `test` actions can define a whether debugger should be used.
```
Comment (yonaskolb) [inline Docs/ProjectSpec.md]: Can you just add `This defaults to true` and then we can merge

Comment (kateinoigakukun) [reply Docs/ProjectSpec.md]: Updated!

**Comment by sassiwalid** [comment]
Hello, but I think that debugEnabled not work, it should be checked and unchecked automatically based on debugEnabled property. I opened this issue https://github.com/yonaskolb/XcodeGen/issues/1000 and I really why when It doesn't work :(

**Pull request merged**